android/src/com/mapswithme/maps/Framework.java

```diff
@@ -20,8 +24,13 @@
   public static final int MAP_STYLE_DARK = 1;
   public static final int MAP_STYLE_CLEAR = 2;
 
+  @Retention(RetentionPolicy.SOURCE)
```

Comment:
Да
> On May 25, 2016, at 7:59 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/Framework.java <https://github.com/mapsme/omim/pull/3340#discussion_r64609441>:
> 
> > @@ -20,8 +24,13 @@
> >    public static final int MAP_STYLE_DARK = 1;
> >    public static final int MAP_STYLE_CLEAR = 2;
> >  
> > +  @Retention(RetentionPolicy.SOURCE)
> А, чтобы при компиляции вырезалось?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3340/files/0eaf8797da6003592602a2144d0bd41203a7c93b..fb35b52ab26a99751d3582d9e6e9c073e8d3b3fe#r64609441>